enarx-keep/tests/integration_tests.rs

```diff
@@ -111,5 +122,15 @@ fn write_stderr() {
 
 #[test]
 fn write_emsgsize() {
-    run_test("write_emsgsize", 0);
+    run_test("write_emsgsize", 0, None);
+}
+
+#[test]
+fn read() {
+    const INPUT: &str = "hello world\n";
+    let child = run_test("read", 0, INPUT);
+    let stdout = child.stdout.unwrap();
+
+    let buf: [u8; INPUT.len()] = read_item(stdout).unwrap();
```

Comment:
There is still one problem with this. Although we know that the string above is 12 bytes, `INPUT.len()` here returns 12 **characters**, which might be more than 12 bytes (but isn't in actuality). We probably actually need something like `INPUT.bytes().len()` but I'm not sure if that will work.